protocols/imap/src/main/java/org/apache/james/imap/message/request/AbstractMessageRangeRequest.java

```diff
@@ -74,6 +74,6 @@ private boolean equals(IdRange[] idSet1, IdRange[] idSet2) {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(idSet, mailboxName, useUids);
+        return Objects.hashCode(Arrays.hashCode(idSet), mailboxName, useUids);
```

Comment:
no equalsverifier?

Reply:
I do not know how to do it on an abstract class